src/main/java/nl/hsac/fitnesse/fixture/util/XMLValidator.java

```diff
@@ -53,4 +65,30 @@ public void isWelformed(String xml) {
             reader.close();
         }
     }
+
+    /**
+     * Validate currently loaded xml against an xsd
+     * @param xsdFileName the XSD schema file to use for validation
+     * @return true if the xml validates against the schema.
+     * Throws a descriptive SlimFixtureException when validation fails
+     */
+    public boolean validateAgainst(String xmlContent, String xsdFileName) {
+        try {
+            String xsdContent = new FileFixture().textIn(xsdFileName);
```

Comment:
I try not to uses fixtures from util classes. Can you move the reading of the file to XmlFixture, and just supply the xsdContent as parameter here.
You could have an overload that reads a file, but that can only use FileUtil.

Reply:
Moved file reading to xmlFixture/xmlHttpTest classes